pkg/cmd/workspace/create.go

```diff
@@ -438,15 +442,36 @@ func processGitURL(repoUrl string, apiClient *apiclient.APIClient, projects *[]a
 	return nil, nil
 }
 
-func waitForDial(tsConn *tsnet.Server, workspaceId string, projectName string) error {
+func waitForDial(workspace *apiclient.Workspace, activeProfile *config.Profile, tsConn *tsnet.Server) error {
+	if workspace.Target == "local" {
+		err := config.EnsureSshConfigEntryAdded(activeProfile.Id, workspace.Id, workspace.Projects[0].Name)
+		if err != nil {
+			return err
+		}
+
+		for {
+			projectHostname := config.GetProjectHostname(activeProfile.Id, workspace.Id, workspace.Projects[0].Name)
+
+			sshCommand := exec.Command("ssh", projectHostname, "daytona", "version")
+			sshCommand.Stdin = nil
+			sshCommand.Stdout = nil
+			sshCommand.Stderr = &util.TraceLogWriter{}
+
+			err = sshCommand.Run()
```

Comment:
Looks like everything before the Run call can go above the `for` loop

Reply:
Removed

Reply:
A command can `Run` only once. Moved the projectHostname init outside the loop